src/main/java/it/pagopa/transactions/repositories/TransactionDocument.java

```diff
@@ -0,0 +1,27 @@
+package it.pagopa.transactions.repositories;
+
+import it.pagopa.ecommerce.commons.domain.TransactionId;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.annotation.PersistenceConstructor;
+import org.springframework.lang.NonNull;
+import org.springframework.lang.Nullable;
+
+public record TransactionDocument(
```

Comment:
What does this document represent? I think wallet data related to a transaction? a more precise name could be used?

Reply:
TransactionCacheInfo